src/utils/id.ts

```diff
@@ -0,0 +1,18 @@
+import dayjs from "dayjs";
+
+export const getPropositionFromId = (propositionId: string) =>
+  propositionId.slice(-2);
+
+export const getMarketDetailsFromId = (marketId: string) => {
+  const timestamp = marketId.slice(0, 6);
```

Comment:
Also... might be worth doing something like 
```suggestion
export const getMarketDetailsFromId = (marketId: string) => {
  if (marketId.length < 9) return marketId;
  
  const timestamp = marketId.slice(0, 6);
```